storage/backup/backup-cli/src/backup_types/state_snapshot/restore.rs

```diff
@@ -131,13 +132,44 @@ impl StateSnapshotRestoreController {
             )
         };
 
-        // FIXME update counters
         ver_gauge.set(self.version as i64);
         tgt_leaf_idx.set(manifest.chunks.last().map_or(0, |c| c.last_idx as i64));
-        for chunk in manifest.chunks {
+        let total_chunks = manifest.chunks.len();
+
+        let resume_point_opt = receiver.previous_key_hash();
+        let chunks = if let Some(resume_point) = resume_point_opt {
+            manifest
+                .chunks
+                .into_iter()
+                .skip_while(|chunk| chunk.last_key <= resume_point)
+                .collect()
+        } else {
+            manifest.chunks
+        };
+        if chunks.len() < total_chunks {
+            info!(
+                chunks_to_add = chunks.len(),
+                total_chunks = total_chunks,
+                "Resumed state snapshot restore."
+            )
```

Comment:
all these infos are going to be really annoying in the aptos CLI, but it's fine for now

Reply:
Better than waiting for an hour without much messaging.
Worse than a text UI with progress bars, but I'm not capable to make it.